encryptor/searchable_query_filter.go

```diff
@@ -56,36 +56,41 @@ func NewSearchableQueryFilter(schemaStore config.TableSchemaStore, mode Searchab
 }
 
 // FilterSearchableComparisons filter search comparisons from statement
-func (filter *SearchableQueryFilter) FilterSearchableComparisons(statement sqlparser.Statement) []SearchableExprItem {
+func (filter *SearchableQueryFilter) FilterSearchableComparisons(statement sqlparser.Statement) map[string]SearchableExprItem {
 	tableExps, err := filter.filterTableExpressions(statement)
 	if err != nil {
 		logrus.Debugln("Unsupported search query")
 		return nil
 	}
 
-	defaultTable, aliasedTables := filter.filterInterestingTables(tableExps)
+	defaultTables, aliasedTables := filter.filterInterestingTables(tableExps)
```

Comment:
actually, it is not default tables. Previously one table was return due to only one table in the `WHERE` statement can be without aliases if somewhere columns addressed without table names or aliases. For example:
`SELECT tbl1Col1, tbl2Col2 FROM table1, table2`. It is invalid query if `tbl1Col1` and `tbl2Col2` are table1's columns. If `tbl2Col2` is column of `table2`, than it is invalid and should be accessed as `table2.tbl2Col2` or using aliases, like `SELECT tbl1Col1, tbl2.tbl2Col2 FROM table1, table2 tbl2`.
Do we handle it now after these changes?